### DISCUSSION TITLE
dns - hostname check

### DISCUSSION THREAD

**Discussion by thezoggy** [sabnzbd/sabnzbd#2576] [General]
duplicati also has a dns rebind protection where you fill out hostname like or host check. they however default it with `*` to disable the feature.

> To prevent various DNS based attacks, Duplicati limits the allowed hostnames to the ones listed here. Direct IP access and localhost is always allowed. Multiple hostnames can be supplied with a semicolon separator. If any of the allowed hostnames is an asterisk (*), all hostnames are allowed and this feature is disabled. If the field is empty, only IP address and localhost access is allowed.

Do we want to support doing that as well (since some people never figure out to just use ip to sidestep it / get frustrated at setting up sab). 
Aslo figure we could move it from config>special to config>general so its less buried. and then borrow their verbiage to better surface users to know to set it.

**Comment by sanderjo** [comment]
What is the use case?
What problem are you describing / solving?

**Reply by thezoggy** [reply]
Just was pointing out what another app does (since I havent really seen others do it that I've come across).
While ideally it shouldn't be a thing as people should* be configuring sab locally / can sidestep it with ip.. but still randomly see people complain that the feature messes them up because they dont understand it / cant decipher what the wiki says/doesnt really show anything in logs.. you just have to kinda know the reason.

**Reply by Safihre** [reply]
Then maybe we should improve the wiki readability? Although it seems pretty straightforward..

**Reply by sanderjo** [reply]
@thezoggy do you mean it's about "SABnzbd Host", where people fill out 127.0.0.1 (good), 0.0.0.0 (good too), or blabla.newsserver.com (bad)?

**Reply by thezoggy** [reply]
> @thezoggy do you mean it's about "SABnzbd Host", where people fill out 127.0.0.1 (good), 0.0.0.0 (good too), or blabla.newsserver.com (bad)?

no our hostname check,
https://sabnzbd.org/wiki/extra/hostname-check.html

which users configure: config>special>host_whitelist

It was just me asking if we should support `*` as a method to disable it. Will close as there is no intrest
